upload-api/tables/upload.js

```diff
@@ -134,13 +144,26 @@ export function useUploadTable(dynamoDb, tableName) {
         throw new Error('Missing `Attributes` property on DynamoDB response')
       }
 
-      return { ok: toUploadAddResult(unmarshall(res.Attributes)) }
+      const raw = unmarshall(res.Attributes)
+      // if new, increment total
+      if (raw.insertedAt === raw.updatedAt) {
+        await Promise.all([
+          metrics.space.incrementTotals({
+            [SPACE_METRICS_NAMES.UPLOAD_ADD_TOTAL]: [{ space, value: 1 }]
+          }),
+          metrics.admin.incrementTotals({
+            [METRICS_NAMES.UPLOAD_ADD_TOTAL]: 1,
+          })
+        ])
```

Comment:
Note: upload metrics now recorded here, instead of via the UCAN stream.

Also more accurate - upload is mutable - we only want to count on the insert, not also update.